entity-framework/ef6/database-initializers.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "🔧 Database Initializers - EF6"
```

Comment:
folks really hate wrench topics. we removed all of them from .NET and ASP.NET docs

Reply:
Thanks for the information. Will talk to the team about tracking this only as issues.